api/bibliotheca.py

```diff
@@ -805,18 +790,47 @@ def value(bibliotheca_key):
             contributors=authors+narrators,
             measurements=measurements,
             links=links,
+            circulation=circulation,
         )
+        return metadata
+
+    def _make_circulation_data(self, tag, namespaces, primary_identifier):
+        """Parse out a CirculationData containing current circulation
+        and formatting information.
+        """
+
+        def value(bibliotheca_key):
+            return self.text_of_subtag(tag, bibliotheca_key)
+
+        def intvalue(key):
+            return self.int_of_subtag(tag, key)
+
+        book_format = value("BookFormat")
+        medium, formats = self.internal_formats(book_format)
 
-        # Also make a CirculationData so we can write the formats,
-        circulationdata = CirculationData(
+        licenses_owned = intvalue("TotalCopies")
+        try:
+            licenses_available = intvalue("AvailableCopies")
+        except IndexError:
+            logging.warn(
+                "No information on available copies for %s",
+                primary_identifier.identifier
+            )
+            licenses_available = 0
+
+        patrons_in_hold_queue = intvalue("OnHoldCount")
+        licenses_reserved = 0
```

Comment:
Are reserved licenses included in one of the other counts then?

Reply:
Yeah, I meant to mention this. Reserved licenses are counted in the hold count.

Bibliotheca is the only provider I've ever seen that exposed 'reserved' books separately from 'on hold' books, and it now appears that was a fluke. The only thing we did with licenses_reserved was keep track of it and add it to the hold count, so I think this means we can get rid of that field altogether.